controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/SealStreamTask.java

```diff
@@ -72,6 +92,43 @@ public SealStreamTask(final StreamMetadataTasks streamMetadataTasks,
                 });
     }
 
+    private CompletableFuture<Boolean> cancelTransactions(OperationContext context, String scope, String stream) {
```

Comment:
You don't want to call it `abortTransactions`? Because that's essentially what we are doing, right?